app/assets/javascripts/statuses.js

```diff
@@ -0,0 +1,62 @@
+$(function() {
+  $(".proxy-version").each(function(index, item) {
+    var item = $(item);
+    var url = item.data('url');
+    $.ajax({
+      type: 'get',
+      url:  url,
+      success: function(response) {
+        item.attr('title',response.message);
+        if(response.success == true) {
+          item.addClass('text-success')
+          item.text(__(response.message));
+        } else {
+          item.addClass('text-danger')
+          item.text(__('Error'));
+        }
+        item.tooltip({html: true});
+      }
+    })
+  });
+
+  $(".proxy-status, .compute-status").each(function(index, item) {
+    var item = $(item);
+    var url = item.data('url');
+    $.ajax({
+      type: 'post',
+      url:  url,
+      success: function(response) {
+        if(response.status == "OK") {
+          item.attr('title', __('Active'));
+          item.addClass('text-success');
+          item.html('<span class="glyphicon glyphicon-ok-sign" />');
+        } else {
+          item.attr('title', response.message);
+          item.addClass('text-danger')
+          item.html('<span class="glyphicon glyphicon-exclamation-sign" />');
+        }
+        item.tooltip({html: true});
+      }
+    })
+  });
+
+  $(".proxy-show-status").each(function(index, item) {
+      var item = $(item);
+      var url = item.data('url');
+      $.ajax({
+        type: 'post',
+        url:  url,
+        success: function(response) {
+          if(response.status == "OK") {
+            item.addClass('text-success');
+            item.html('<span class="glyphicon glyphicon-ok-sign" />  ' + __('Active'));
+          } else {
+            item.attr('title', response.message);
+            item.addClass('text-danger')
+            item.html('<span class="glyphicon glyphicon-exclamation-sign" />  ' + __('Error'));
+          }
+          item.tooltip({html: true});
+        }
+      })
+    });
+});
```

Comment:
This three methods start looking as repeating, can we extract some bits into helper function? I expect more statuse fields to be added soon.